release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1559,6 +1559,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* When adding a Git Repository, the created repository resource was invalid if you configured it with a GitLab and Bitbucket `pipeline-as-a-code` repository. Now, the Git provider URL is removed for GitLab and Bitbucket providers. A mandatory *Bitbucket username* field is added if the provider is Bitbucket. You also need to select either *Git access token* or *Git access token secret* to continue adding a Git Repository. (link:https://issues.redhat.com/browse/OCPBUGS-7036[*OCPBUGS-7036*])
```

Comment:
@shreyasiddhartha I see that the issue here is that the created repository resource was invalid. Are the remaining sentences the workaround for this issue? Or they are also part of the Known issue?

Also, I do not see any mention of this: spec.git_provider.user details for BitBucket provider. Any particular reason why this is not mentioned?

Reply:
@shreyasiddhartha Also please note that this is the template RH SSG suggests for writing the RN Texts: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts

Please adhere to these guidelines and standards when documenting RN Texts for KIs or KIs with workarounds. Thanks!

Reply:
@Srivaralakshmi I have made a few changes as per your suggestions. PTAL. Thanks!